google/ggrc-core
# GGRC-6590 Replace Related Objectives w/ Related Requirements section for Control snapshot

Should be merged in ```2.0.0``` release (3/21)

# Issue description

Related Requirements section should be shown instead of Objectives on info pane for Control snapshot.

# Steps to test the changes

 - Create Requirement, Regulation & Control objects
 - Map created Requirement & Regulation to Control
 - Create Audit & map created Requirement, Regulation & Control to it
 - Create Assessment -> Map Control to it
 - Open Assessment info pane
 - See 'CONTROLS' section with attached control -> Click 'More Information'
 - Scroll down the popup

Actual result: The following sections are shown 'Show Related Regulations', 'Show Related Objectives'
Expected result: The following sections should be shown 'Show Related Requirements' (NEW!), 'Show Related Regulations'

# Solution description

Replace ```Objectives``` section to ```Requirements``` in **<assessment-mapped-controls>** template. Add appropriate params to request for related objects.

# Sanity checklist




# PR Review checklist
